code/modules/projectiles/guns/projectile/pistol.dm

```diff
@@ -74,3 +74,9 @@
 		user << "<span class='notice'>..and falls into view. Whew, that was a close one.</span>"
 		user.unEquip(src)
 
+/obj/item/weapon/gun/projectile/automatic/pistol/mgs

```

Comment:
you should have the name referencing what the object does or it's name [ie pistol/sleepy or pistol/hushpuppy], not the game reference.